src/main/java/seedu/address/model/person/Person.java

```diff
@@ -31,6 +31,7 @@
     private final JobTitle jobTitle;
     private final HashSet<Pronoun> pronouns = new HashSet<>();
     private final HashSet<Tag> tags = new HashSet<>();
+    //private final HashSet<Meeting> meetings = new HashSet<>();
```

Comment:
I am not really sure why this is commented out

Reply:
We don't need meetings associated with each person right, since they are loaded dynamically? I think we can safely remove this.

Reply:
I believe we were targeting displaying the meetings associated with each person. So when you issue the view command it just iterates through the meetings list to find meetings involving that particular contact.